platform/ui/src/components/index.js

```diff
@@ -9,6 +9,7 @@ import Dialog from './Dialog';
 import Dropdown from './Dropdown';
 import EmptyStudies from './EmptyStudies';
 import ErrorBoundary from './ErrorBoundary';
+import GenerateVolume from './GenerateVolume/GenerateVolume';
```

Comment:
this GenerateVolume is toooooo specific to be included in the platform/ui, move it to the extension please